hips/draw/simple.py

```diff
@@ -178,11 +179,48 @@ def draw_tiles(self) -> np.ndarray:
 
         return image
 
+    def draw_hips_tile_grid(self) -> None:
```

Comment:
Contrary to what I said before, this isn't a good name for this method.
It doesn't just draw the tile grid, it also draws the sky iamge.

So my suggestion would be to rename to "draw_debug_image", and maybe even to split out the two parts into `_draw_sky_image` and `_draw_hips_tile_grid` functions and call the two from `draw_debug_image`?